### PR TITLE
MAIN-5217 revert hook code which was causing CreateNewWiki process to fail

### PR DESCRIPTION
**Pull request by jsutterfield**
The reverted hooks were calling `$importer->nodeContents();` which was advancing a file pointer on the `/dumps/starter.xml.bz2` file. When that hook exited and control was given back to `WikiImporter`, `WikiImporter` didn't realize that the pointer had been advanced and was then reading incorrect values which led to an error.

Ticket: https://wikia-inc.atlassian.net/browse/MAIN-5127

Related PRs:
https://github.com/Wikia/app/pull/8063
https://github.com/Wikia/app/pull/8062

**EDIT**: This PR had a typo in it when it was merged. It's fixed it in this commit: https://github.com/Wikia/app/commit/a8b58a415fa5669d1ae0b16cd1b4a690a68d1618

### CODE REVIEW

File: includes/Import.php [hunk 1]
```diff
@@ -819,6 +814,9 @@ private function processTitle( $text ) {
 			# Do not import if the importing wiki user cannot create this page
 			$this->notice( 'import-error-create', $title->getPrefixedText() );
 			return false;
+		} elseif( $title->getNamespace() == NS_MEDAWIKI  ) {
+			# Do not import if pages in the MediaWikia Namespace
+			return false;
```
Comment (jsutterfield) [inline includes/Import.php:820]: This prevents importing pages from the MW namespace which was the intent MAIN-5118 which was reverted.

Comment (owend) [inline includes/Import.php:820]: Ah, this solution looks like a better approach.

**Comment by owend** [comment]
This LGTM

**Comment by jsutterfield** [comment]
Had a typo in this PR. fixed it in this commit: https://github.com/Wikia/app/commit/a8b58a415fa5669d1ae0b16cd1b4a690a68d1618

**Pull request merged**